people/methods/get.js

```diff
@@ -0,0 +1,23 @@
+var mysql = require('mysql');
+
+const con = mysql.createConnection({
+  host: "localhost",
+  user: "root",
+  password: "123456",
+  database: "interns"
+});
+
+con.connect(function (err) {
```

Comment:
this is not as per the sample application please fix it this is creating the connection every time a request is being hit